### ISSUE TITLE
Support for Factorio 2.0

### ISSUE THREAD

**Issue by gth** [dbeckwith/factorio-infinite-resources#3]
Anything we can do to help bump this mod to support F_2.0 ?   Happy to help if I can...

**Comment by gth** [comment]
I manually bumped the version number and seems to be working fine.  I haven't done a github pull request before, but the simple changes I made were (after unpacking the relevant zip file from the mods folder):

changelog.txt, insert at top of file:
```
---------------------------------------------------------------------------------------------------
Version: 0.4.9
Date: 2024-10-22
  Minor Changes:
    - Bumped Factorio version number
```

info.json, modify the lines:
replace:
`  "version": "0.4.8",`
with:
`  "version": "0.4.9",`

and replace:
`  "factorio_version": "1.1",`
with:
`  "factorio_version": "2.0",`

Renamed the folder `sonaxaton-infinite-resources_0.4.9`, made it a standard zip file, swapped out the old zip with the new one in `%APDATA%\Factorio\mods\`

Restarted Factorio and voila - Infinite Resources mod is now compatible with Factorio 2.0.  :)
The ore patches and oil all show 100%, so I think it's all good.
